setup.py

```diff
@@ -226,14 +226,21 @@ def use_bundled_luajit(path, macros):
     print('Building LuaJIT for %r in %s' % (platform, libname))
 
     src_dir = os.path.join(path, "src")
+
+    build_env = dict(os.environ)
     if platform.startswith('win'):
         build_script = [os.path.join(src_dir, "msvcbuild.bat"), "static"]
         lib_file = "lua51.lib"
     else:
         build_script = ["make",  "libluajit.a"]
         lib_file = "libluajit.a"
 
-    output = subprocess.check_output(build_script, cwd=src_dir)
+        if 'CFLAGS' in build_env:
+            build_env['CFLAGS'] += " -fPIC"
```

Comment:
Most people who have a `CFLAGS` variable set probably have their CFLAGS properly configured for their system already.
```suggestion
        if 'CFLAGS' in build_env:
            if "-fPIC" not in build_env['CFLAGS']:
                build_env['CFLAGS'] += " -fPIC"
```